core/domain/suggestion_jobs_one_off.py

```diff
@@ -348,3 +348,45 @@ def reduce(key, values):
             yield (key, len(values))
         else:
             yield (key, values)
+
+
+class ContentSuggestionFormatUpdateJob(jobs.BaseMapReduceOneOffJobManager):
+    """Job that migrates the format of content suggestions."""
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [suggestion_models.GeneralSuggestionModel]
+
+    @staticmethod
+    def map(item):
+        if item.suggestion_type != feconf.SUGGESTION_TYPE_EDIT_STATE_CONTENT:
+            return
+
+        changes_made = False
+        if 'old_value' in item.change_cmd:
+            del item.change_cmd['old_value']
+            changes_made = True
+            yield ('CHANGED - Removed old_value', item.id)
+        if 'content_id' in item.change_cmd['new_value']:
+            del item.change_cmd['new_value']['content_id']
+            changes_made = True
+            yield ('CHANGED - Removed content_id', item.id)
+
+        # Validate the eventual suggestion format.
+        try:
+            assert set(item.change_cmd.keys()) == set([
+                'state_name', 'cmd', 'new_value', 'property_name'])
+            assert item.change_cmd['cmd'] == 'edit_state_property'
+            assert item.change_cmd['property_name'] == 'content'
+            assert item.change_cmd['new_value'].keys() == ['html']
+        except AssertionError as e:
+            yield ('Failed assertion', '%s %s' % (item.id, e))
+
+        if changes_made:
+            item.update_timestamps(update_last_updated_time=False)
```

Comment:
Are we expecting this function call to update any timestamps? (This will update the created_on if it's none.)